tests/test_output_formatter.cpp

```diff
@@ -318,6 +318,33 @@ auto construct_find_multiple_reply_duplicate_image()
 
     return reply;
 }
+
+auto construct_version_info_multipassd_update_available()
+{
+    auto reply = mp::VersionReply();
+    reply.set_version("Daemon version");
+    reply.set_log_line("Some logline goes here");
+
+    mp::UpdateInfo* updateInfo = new mp::UpdateInfo; // Leak? expecting protobuf to deal with memory dealloc.
```

Comment:
There's `reply.mutable_update_info()` that you can modify directly to avoid the leak, like here:

https://github.com/canonical/multipass/blob/c35bfe38535f6cc6b20b4effaefbc9a62b10846a/src/daemon/daemon.cpp#L1989-L1991